packages/ui5-middleware-simpleproxy/lib/proxy.js

```diff
@@ -145,10 +145,17 @@ module.exports = function ({ resources, options }) {
     https: protocol === "https",
     limit: limit,
     filter: excludePatterns && Array.isArray(excludePatterns) && function(req, res) {
-      return excludePatterns.some(glob => {
+      const pattern = !excludePatterns.some(glob => {
         isDebug && log.info(`Proxy request ${req.url} is matched to glob "${glob}": ${minimatch(req.url, glob)} ${providedBaseUri}`);
-        return !minimatch(req.url, glob);
+
+        return minimatch(req.url, glob);
       });
+      if (!pattern && req.url.includes("~") && (options.configuration && options.configuration.skipCache)) {
+        const newUrl = req.url.slice(0,req.url.indexOf("~")) + req.url.slice(req.url.lastIndexOf("~") + 3,req.url.length)
+        isDebug && log.info(`Removing cache from ${req.url}, resolving to ${newUrl}`);
+        req.url = newUrl
+      }
+      return pattern
```

Comment:
I need to add prettier to enforce code conventions (Semicolon or not semicolon => that's the question!)... 😜

Reply:
I've added semicolons as I could see that was the convention. I prefer not to have them though ;)

If i did a prettier, it would make changes to the entire file. Maybe add lint-staged and a seperate package with the prettier config?

Reply:
👍 - we could do that in a separate change - in general, I think using prettier in one or the other direction may be the best option and enforce it with a commit hook or so to come to "one" style when many people are working together on one project... If you are OK with going for it later then we submit this one and do the prettier stuff independently?

Reply:
yeah sounds good. I'll open an issue. We can probably get many opinions on the prettier config as well :crossed_swords: